repo/packages/D/datadog/0/marathon.json

```diff
@@ -0,0 +1,59 @@
+{
+    "id": "datadog-agent",
```

Comment:
we need this to be configurable... it is commonly framework-name, however this doesn't seem to be a framework.  We still need this to be a configurable {{data-dog.app-id}}